build-support/functions/20-build.sh

```diff
@@ -464,7 +464,11 @@ function build_consul_local {
             then
                GOBIN_EXTRA="${os}_${arch}/"
             fi
-            CGO_ENABLED=0 GOOS=${os} GOARCH=${arch} go install -ldflags "${GOLDFLAGS}" -tags "${GOTAGS}" && cp "${MAIN_GOPATH}/bin/${GOBIN_EXTRA}consul" "${outdir}/consul"
+            binname="consul"
+            if [ $os == "windows" ];then
+                binname="consul.exe"
+            fi
+            CGO_ENABLED=0 GOOS=${os} GOARCH=${arch} go install -ldflags "${GOLDFLAGS}" -tags "${GOTAGS}" && cp "${MAIN_GOPATH}/bin/${GOBIN_EXTRA}$binname" "${outdir}/$binname"
```

Comment:
Its a purely stylistic thing but I always enclose variables in `{}`

Reply:
Okay.